src/main/java/beans/CurrentResponseBean.java

```diff
@@ -0,0 +1,67 @@
+package beans;
+
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonSetter;
+import org.json.JSONArray;
+import session.FormEntrySession;
+
+import java.util.Map;
+
+/**
+ * Created by willpride on 1/20/16.
+ */
+public class CurrentResponseBean {
+    private JSONArray tree;
+    private String title;
+    private String[] langs;
+    private String sessionId;
+
+    public CurrentResponseBean(){
```

Comment:
The library that handles mapping these objects to and from JSON (Jackson) requires a default constructor